ui/app/AppLayouts/Wallet/panels/BuyReceiveBanner.qml

```diff
@@ -0,0 +1,80 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import AppLayouts.Wallet.controls 1.0
+
+import StatusQ.Core.Theme 0.1
+
+Control {
+    id: root
+
+    property bool closeEnabled: true
+    property bool buyEnabled: true
+    property bool receiveEnabled: true
+
+    readonly property bool anyVisibleItems: buyCard.visible || receiveCard.visible
+
+    signal buyClicked()
+    signal receiveClicked()
+    signal closeBuy()
+    signal closeReceive()
+
+    contentItem: RowLayout {
+        id: layout
+        spacing: 16
+
+        BannerCard {
+            id: buyCard
+            objectName: "buyCard"
+            Layout.fillWidth: true
+            title: qsTr("Ways to buy crypto")
+            subTitle: qsTr("Via card or bank transfer")
+            image: Theme.png("wallet/wallet-green")
+            closeEnabled: root.closeEnabled
+            visible: implicitWidth > 0
+            implicitWidth: opacity > 0 ? layout.width / 2 : 0
```

Comment:
`implicitWidth` in a layout?

Reply:
Ah.... because of the `Behavior` :thinking:

Reply:
> Ah.... because of the `Behavior` 🤔

Not just that. But the layout won't properly split the available space into 2 equal parts. The first one would be equal to the implicitWidth and the other one will fill the available space. But we don't want that.

Reply:
not working..

The challenge here is to accommodate all these things:
1. Have an automatic layout (preferable, not required 😄 )
2. Splitting the banner into equal parts
3. Animating width when closing the cards

The only working solution I could see here is to manipulate the `implicitWidth`.

Maybe another option would be to drop the Layout in favor of anchors. 🤔

Reply:
Ok, you were onto something 🍻 

Works with something like this:
`Layout.preferredWidth: root.buyEnabled ? layout.width / layout.children.length : 0`

Updated. Please have a look